assets/models/system/Mistral-7B-Instruct-v0-1/spec.yaml

```diff
@@ -11,6 +11,7 @@ tags:
   Featured: ""
   Preview: ""
   SharedComputeCapacityEnabled: ""
+  MaaS-inference: true
```

Comment:
lets update to all lower case as shared by @skasturi

Reply:
Done